backend/app/services/execution/schedule_helper.py

```
@@ -289,6 +330,36 @@ async def _dispatch_task_async(task_id: int) -> None:
+    treated as "pod still alive" so dispatch surfaces the real error instead
+    of triggering a spurious recovery.
+    """
+    from app.services.remote_workspace_service import remote_workspace_service
+
+    try:
+        alive = await asyncio.to_thread(
+            remote_workspace_service.executor_alive,
+            executor_name,
+            executor_namespace,
+        )
+    except Exception as exc:
```

Ruff (0.16.1)

[warning] Do not catch blind exception: `Exception`

(BLE001)